### PR TITLE
pr-fix timestamps

### LINKED ISSUE #231: Go through all proto files and make sure timestamps are 64 bits
Just stumbled across inconsistencies.

### PR DESCRIPTION
**Pull request by EliaTarasov**
This wip is intended to fix [timestamp issue](https://github.com/PX4/sitl_gazebo/issues/231)

### CODE REVIEW

**Comment by TSC21** [comment]
Is it still WIP or ready yo review?

**Comment by EliaTarasov** [comment]
Tested with:
GPS
Optical Flow
Lidar
Odometry
Wind
Not tested with:
Groundtruth
IRLock

File: src/gazebo_opticalflow_plugin.cpp [hunk 1]
```diff
@@ -79,6 +79,12 @@ void OpticalFlowPlugin::Load(sensors::SensorPtr _sensor, sdf::ElementPtr _sdf)
     return;
   }
 
+#if GAZEBO_MAJOR_VERSION >= 7
```
Comment (TSC21) [inline src/gazebo_opticalflow_plugin.cpp]: here as well

**Comment by EliaTarasov** [comment]
@TSC21 i will remove last commit. My bad.

**Comment by TSC21** [comment]
https://travis-ci.org/PX4/sitl_gazebo/jobs/439632682#L1156 @EliaTarasov you probably need to keep some conditions so it can build with Gazebo 8.

**Review by TSC21 — APPROVED** [review]

**Pull request merged**